### PR TITLE
add csilvm pv metrics, persist other unsaved changes

### PR DESCRIPTION
**Pull request by gpaul**
Fixes https://jira.mesosphere.com/browse/DCOS-55107

### CODE REVIEW

**Review by bbannier — APPROVED** [review]
A little hard to see exactly what happened here, but makes sense to me. Yee-haw!

File: dashboards/Storage-Summary.json [hunk 1]
```diff
@@ -811,6 +769,120 @@
...
+      "title": "Gauge for LVM physical volume statistics",
+      "transparent": false,
+      "type": "text"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
```
Comment (philipnrmn) [inline dashboards/Storage-Summary.json]:
```suggestion
      "datasource": "${DS_PROMETHEUS}",
```

File: dashboards/Storage-Summary.json [hunk 2]
```diff
@@ -920,318 +992,317 @@
...
+      "panels": [],
+      "title": "Operational behavior",
+      "type": "row"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
```
Comment (philipnrmn) [inline dashboards/Storage-Summary.json]:
```suggestion
      "datasource": "${DS_PROMETHEUS}",
```

File: dashboards/Storage-Summary.json [hunk 3]
```diff
@@ -920,318 +992,317 @@
...
+      "mode": "markdown",
+      "title": "Rate of some events from the Mesos master",
+      "type": "text"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
```
Comment (philipnrmn) [inline dashboards/Storage-Summary.json]:
```suggestion
      "datasource": "${DS_PROMETHEUS}",
```

File: dashboards/Storage-Summary.json [hunk 4]
```diff
@@ -920,318 +992,317 @@
...
+      "mode": "markdown",
+      "title": "Size of the service ACK queue",
+      "type": "text"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
```
Comment (philipnrmn) [inline dashboards/Storage-Summary.json]:
```suggestion
      "datasource": "${DS_PROMETHEUS}",
```

**Comment by philipnrmn** [comment]
Exporting the dashboard via the 'share->export' dialog and ticking the 'export for sharing externally' box will fix the datasource issue:

image

**Review by philipnrmn — CHANGES_REQUESTED** [review]
(otherwise, lgtm)

**Comment by gpaul** [comment]
@philipnrmn thanks! Good spot! Also, that export trick is exactly what i was looking for. Unfortunately it didn't _quite_ work as we appear to have removed a bunch of alerts which I'm not sure we really wanted to remove. Anyway, I've fixed the datasource fields, thanks.

**Review by philipnrmn — APPROVED** [review]

**Pull request merged**